packages/next/client/image.tsx

```diff
@@ -238,7 +238,7 @@ function getInt(x: unknown): number | undefined {
 }
 
 export default function Image({
-  src,
+  src = '',
```

Comment:
```suggestion
  src,
```

This doesn't handle the `null` case.